src/main/java/dev/tycho/stonks/command/company/CreateServiceCommandSub.java

```diff
@@ -0,0 +1,46 @@
+package dev.tycho.stonks.command.company;
+
+import dev.tycho.stonks.command.base.CommandSub;
+import dev.tycho.stonks.gui.AccountSelectorGui;
+import dev.tycho.stonks.gui.CompanySelectorGui;
+import dev.tycho.stonks.managers.DatabaseHelper;
+import dev.tycho.stonks.model.core.Company;
+import org.bukkit.ChatColor;
+import org.bukkit.command.CommandSender;
+import org.bukkit.entity.Player;
+
+import java.util.List;
+
+public class CreateServiceCommandSub extends CommandSub {
+  @Override
+  public List<String> onTabComplete(CommandSender sender, String alias, String[] args) {
+    return null;
+  }
+
+  @Override
+  public void onCommand(Player player, String alias, String[] args) {
+    if (args.length > 4) {
```

Comment:
Stop. Making. Large. If Statements. Use the same args checking I've used in every other CommandSub.

Reply:
https://github.com/schlatt-co/stonks/blob/a34c87a8a16fdce21ba0462044aedb07a35af01b/src/main/java/dev/tycho/stonks/command/company/CreateServiceCommandSub.java#L50-L66